GitCommands/Git/ExecutableExtensions.cs

```diff
@@ -16,7 +16,7 @@ namespace GitCommands
     public static class ExecutableExtensions
     {
         private static readonly Regex _ansiCodePattern = new Regex(@"\u001B[\u0040-\u005F].*?[\u0040-\u007E]", RegexOptions.Compiled);
-        private static readonly Encoding _defaultOutputEncoding = GitModule.SystemEncoding;
+        private static readonly Lazy<Encoding> _defaultOutputEncoding = new Lazy<Encoding>(() => GitModule.SystemEncoding, false);
```

Comment:
If we don't make it lazy, it gets executed and that has a snowball effect - a hardcoded instance of `GitModule` gets created, that executes a git command to determine a default system encoding...